hunter2:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  

  # Velocity Controllers ---------------------------------------
  wheel_joint1_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_joint1
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

  wheel_joint2_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_joint2
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

  wheel_joint3_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_joint3
    pid: {p: 0.5, i: 0.0, d: 0.0, i_clamp: 0.0}

  wheel_joint4_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_joint4
    pid: {p: 0.5, i: 0.0, d: 0.0, i_clamp: 0.0}


  # Position Controllers ---------------------------------------
  wheel_joint3_steering_position_controller:
    joint: wheel_joint3_steering
    type: effort_controllers/JointPositionController
    pid: {p: 2.0, i: 0.01, d: 0.5}
  wheel_joint4_steering_position_controller:
    joint: wheel_joint4_steering
    type: effort_controllers/JointPositionController
    pid: {p: 2.0, i: 0.01, d: 0.5}

